.github/workflows/container_images.yaml

```diff
@@ -0,0 +1,94 @@
+on:
+  pull_request:
+  workflow_call:
+    inputs:
+      image-tag:
+        type: string
+  workflow_dispatch:
+    inputs:
+      image-tag:
+        type: string
+
+name: 'Container images'
+
+jobs:
+  determine-image-tags:
+    name: 'Determine image tags'
+    outputs:
+      tags: ${{ steps.determine-image-tags.outputs.tags }}
+    runs-on: ubuntu-latest
+    steps:
+      - id: determine-image-tags
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+          PR_NUMBER: ${{ github.event.number }}
+          CUSTOM_TAG: ${{ inputs.image-tag }}
+        run: |
+          tags=()
+
+          if [[ "$EVENT_NAME" == "pull_request" ]]; then
+            tags+=("pr-$PR_NUMBER")
+          fi
+
+          if [[ "$IMAGE_TAG" != "" ]]; then
+            tags+=("$IMAGE_TAG")
+          fi
+
+          tags_out=""
+
+          for tag in "${tags[@]}"; do
+            if [ -z "$tags_out" ]; then
+              tags_out="\"$tag\""
+            else
+              tags_out="$tags_out,\"$tag\""
+            fi
+          done
+
+          echo "TAGS: $tags_out"
+
+          echo -n "tags=[$tags_out]" >> $GITHUB_OUTPUT
+
+  build-and-push:
+    name: 'Build and push'
+    runs-on: ubuntu-latest
+    needs: ['determine-image-tags']
+    concurrency: 'docker-build'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    strategy:
+      matrix:
+        image: [{ name: 'orb', file: 'images/orb/Dockerfile' }]
```

Comment:
Add an entry to this JSON-y value to include another container build in the workflow